app/src/main/java/com/ncnf/user/FirebaseUserModule.java

```diff
@@ -0,0 +1,19 @@
+package com.ncnf.user;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.hilt.InstallIn;
+import dagger.hilt.android.components.ActivityComponent;
+
+@Module
+@InstallIn(ActivityComponent.class)
+public class FirebaseUserModule {
+
+    @Provides
+    public static FirebaseUser getFirebaseUser() {
```

Comment:
This can then be injected at any point ? Nice to have this, thus avoiding Firebase code in every class

Reply:
Yes you can. I created this because I do not need to load an user from the database to display the friends, I just need the uuid of the current user, hence I only need the FirebaseUser.

Reply:
I think this is the way we will use every repository once implemented ! Much easier